docs/developer/checkout/address.mdx

```diff
@@ -229,6 +229,14 @@ As a result, we get an updated checkout object with a delivery method set:
 }
 ```
 
+### Switching from C&C to standard delivery method
+
+After choosing the click & collect option on a checkout, going back to a standard delivery method will require the shipping address to be set, this can't be done directly by checkoutDeliveryMethodUpdate, but can be achieved using the flow below.
+
+1. First use `checkoutDeliveryMethodUpdate` (just with `id` or `token` provided). This will remove the collection point from checkout and also remove the shipping address.
+2. Now `checkoutAddressUpdate` can be used to set the new shipping address for checkout.
```

Comment:
It's not a migration that should be used. The valid migration is `checkoutShippingAddressUpdate`. We should also add a link to this migration

Reply:
Mutation 😄 but I agree with Iga's